### ISSUE TITLE
Better visibility of notifications - feature request

### ISSUE THREAD

**Issue by psuplat** [uikit/uikit#3540]
I feel that the notification in UIkit are a bit too blended with the background, loosing their impact, especially if we are talking warning or danger states.

Currently I am simply extending the css and swapping the colors to make them more pronounced, but maybe it would be a good idea to add another class/option to the framework. Something like this maybe:

```
/** hivis for HiVisibility**/
/*
 * Primary
 */
.uk-notification-message-primary-hivis {
	color: #fff;
	background-color: #1e87f0;
}
/*
 * Success
 */
.uk-notification-message-success-hivis {
	color: #fff;
	background-color: #32d296;
}
/*
 * Warning
 */
.uk-notification-message-hivis {
	color: #fff;
	background-color: #faa05a;
}
/*
 * Danger
 */
.uk-notification-message-hivis {
	color: #fff;
	background-color: #f0506e;
}
```

The result is like this:
![notifications inverted](https://user-images.githubusercontent.com/14207860/44313977-f791f980-a409-11e8-8029-a1d54bff73b9.png)

What do you guys think?

**Comment by drawcard** [comment]
It would be good to have this as a second option for accessibility purposes, as the original design is still nice but doesn't have the same kind of colour contrast.

**Comment by psuplat** [comment]
How about this:

[http://jsfiddle.net/v9tkw5dz/15/](url)

You get a hivis option and a bordered version.
The bordered version is more toned and in line with the original design, but still stands out better if it was to show agains, let say a navbar element.

**Comment by tyler-x** [comment]
I like the hivis option!

Agreed, the current notification are sometimes hard to notice. Use colored background will be a better solution, and also keep the current mute color background as an option or default would be great.

**Comment by razvanioan** [comment]
+1

**Comment by jsieler** [comment]
The same style could be made for every element on uikit, for example buttons too.... but they don't want.

**Comment by szabeszg** [comment]
> but they don't want.

:(

**Comment by jsieler** [comment]
See: https://github.com/uikit/uikit/issues/2680

**Comment by zzseba78** [comment]
@psuplat I agree, in fact i have this implemented in the notification less file,  the default style just change color text, it's almost not noticeable.

Agree with @jsieler too, it should be some consistency in the status styling, for example:  Background, Badge, Button, Form... I think it´s a way of keep UIkit as light as possible, but i think this should be covered. 

### Status as a separate component

Create a Status component where you can choose if use or not status modifiers ( Success, Warning, Danger ) globally or for each component: Alert, Background, Badge, Button, Form,  Icons, Label, Marker, Notification, Progress, Text. Why to load all the styles if i will use Status only on buttons for example.

### Global Modifiers

Simplify the styling using global modifiers: for example a button that it´s success, just add `uk-success` and it´s done. All the same styles could be apply to almost all the elements, for example a success color background and white text could be appied on: Alert, Background, Badge, Button,    Label, Marker, Notification, Progress.

I don´t know, i just some ideas.

**Comment by jsieler** [comment]
@drawcard if you build some government website, you will probably re-style all elements anyway.

**Comment by drawcard** [comment]
Probably - but it would be nice if the defaults were at least a bit more readable than they currently are.

**Comment by xpallicer** [comment]
I'm sorry to say this, but default notification style in uikit3 still sucks on aug 19

**Comment by AlexandreZaytsev** [comment]
wrapping styles from [alert](https://getuikit.com/assets/uikit/tests/alert.html) messages

```
/*
 * Primary
 */
.uk-notification-message-primary{
    background: #d8eafc;
    color: #1e87f0;
}
/*
 * Success
 */
.uk-notification-message-success {
   background: #edfbf6;
   color: #32d296;
}
/*
 * Warning
 */
.uk-notification-message-warning {
    background: #fff6ee;
    color: #faa05a;
}
/*
 * Danger
 */
.uk-notification-message-danger {
    background: #fef4f6;
    color: #f0506e;
}
.uk-notification-message{
/*	color: #fff;*/
  border-radius: 10px;
}

```
